solc/CommandLineInterface.cpp

```diff
@@ -63,6 +63,7 @@ static string const g_argAsmJsonStr = "asm-json";
 static string const g_argAstStr = "ast";
 static string const g_argAstJson = "ast-json";
 static string const g_argBinaryStr = "bin";
+static string const g_argRuntimeBinaryStr = "bin-runtime";
```

Comment:
MAybe rename this to "runtime-bin" to be consistent with the "clone-bin" argument?